_sass/components/_library-card.scss

```diff
@@ -0,0 +1,78 @@
+// Library card ------------- //
+
+// Sets a height on library entries so they're the same height and don't break the grid
+.library-card {
+  box-shadow: 0 0 0 1px $color-gray-lighter;
+  display: block;
+  margin-bottom: 3rem;
+  padding-bottom: 3rem;
+  position: relative;
+  text-decoration: none;
+
+  @include media($medium-screen) {
+    display: flex;
+    flex-direction: column;
+    margin-right: 2.3%;
+    min-height: 350px;
```

Comment:
@line47 for the sake of time, do you mind making the edits yourself instead of as comments?